projects/1.19/assets/playerrevive/playerrevive/lang/zh_cn.json

```
@@ -0,0 +1,8 @@
+{
+	"playerrevive.chat.bleeding": "%s 伤重垂危…… %s",
+	"death.attack.bledToDeath.player": "%s 流血致死",
+	"playerrevive.gui.button.send": "发送",
+	"playerrevive.gui.label.time_left": "剩余时间：%s 秒",

```

```suggestion
	"playerrevive.gui.label.time_left": "剩余时间：%s 秒",
```
请求确认占位符的内容，万一它会是“1分2秒”这个格式就不好了